embed.py

```diff
@@ -0,0 +1,47 @@
+#import librosa
+import torch
+from WavLM import WavLM, WavLMConfig
+
+#audio_name = "testaudio.wav"
+
+#decode audio with librosa:
+#waveform1, sr = librosa.load(audio_name)
+
+#decode audio with faster-whisper:
+#sr1 = 16000
+#waveform1 = decode_audio(audio_name, sr1)
+
+def load_model():
+  '''
+  Loads feature extraction model, configuration and weights.
+  '''
+  # load the pre-trained checkpoints
+  model_checkpoint_loc = 'WavLM-Base+.pt' #WavLM-Large consumed way too much RAM
```

Comment:
This code will be running at some point in a DANE worker, right? I would imagine there will be more resources available to run a larger version of WavLM, so it would make sense to change the checkpoint to `WavLM-Large`. Maybe Sara can comment more about this.